bioblend/_tests/TestGalaxyWorkflows.py

```diff
@@ -167,6 +167,30 @@ def test_run_workflow(self):
         with self.assertRaises(Exception):
             self.gi.workflows.run_workflow(wf['id'], None)
 
+    def test_invoke_workflow(self):
```

Comment:
Can you please add `@test_util.skip_unless_galaxy('release_15.03')` above this line?